mesh/src/common/MutableElement.cpp

```diff
@@ -370,7 +370,6 @@ MutableElement<1, SPACE_DIM>::MutableElement(unsigned index, const std::vector<N
     mEdgeHelper(nullptr)
 {
     // Sanity checking
-    assert(this->mNodes.size() == 2);
```

Comment:
Is this on purpose, the immersed boundary has a mutable 1d element with only 1 node in it does it?

Reply:
Yes, in 1D element in 2D spaces are used to represent laminas. These have variable numbers of nodes depending on their length and node density.